libs/deal-ticket/src/components/trading-mode-tooltip/compile-grid-data.tsx

```diff
@@ -60,10 +60,11 @@ export const compileGridData = (
   if (isLiquidityMonitoringAuction && market.data?.suppliedStake) {
     grid.push({
       label: (
-        <Link href={`/liquidity/${market.id}`} passHref={true}>
-          <UiToolkitLink onClick={() => onSelect && onSelect(market.id)}>
-            {t('Current liquidity')}
-          </UiToolkitLink>
+        <Link
+          to={`/liquidity/${market.id}`}
```

Comment:
shouldn't this one be '#/' too?

Reply:
No react router handles the hash character as we are using the HashRouter provider